src/selectors/temperatureSync.js

```diff
@@ -47,11 +47,13 @@ export const selectTemperatureSyncIsComplete = ({ temperatureSync }) => {
 };
 
 export const selectTemperatureSyncStateMessage = ({ temperatureSync }) => {
-  const { syncState, syncError } = temperatureSync;
+  const { syncState, syncError, lastTemperatureSync } = temperatureSync;
+  const formattedDate = moment(lastTemperatureSync).format('D.M.YYYY');
 
-  return (syncError && syncStrings.sync_error) || syncState
-    ? syncStrings.sync_in_progress
-    : syncStrings.sync_enabled;
+  return (
+    (syncError && `${syncStrings.sync_error}. ${syncStrings.last_sync} ${formattedDate}`) ||
+    (syncState ? syncStrings.sync_in_progress : syncStrings.sync_enabled)
```

Comment:
Hm, feel like this would be a bit clearer not as a one-liner?
 
```suggestion
    if (syncError) return `${syncStrings.sync_error}. ${syncStrings.last_sync} ${formattedDate}`;
    return syncState ? `${syncStrings.sync_error}. ${syncStrings.last_sync} ${formattedDate}`;
```

Reply:
9e1ad53